sources/strapi/README.md

```diff
@@ -1,109 +1,52 @@
----
-title: Strapi
-description: dlt source for Strapi API
-keywords: [strapi api, strapi source, strapi]
----
-
 # Strapi
 
 Strapi is a headless CMS (Content Management System) that allows developers to create powerful API-driven content management systems without having to write a lot of custom code.
 
-Since the endpoints that will be available in Strapi depend on your setup, in order to get data from strapi to your warehouse, you need to be aware of which endpoints you will ingest.
-
-## Grab API token
-
-1. Log in to your Strapi account
-2. Click on `⚙️ settings` in the left-hand sidebar menu
-3. In the settings menu, go to the `API tokens` option that is present under global settings
-4. Click on `Create new API token`
-5. Fill in the details like `Name`, `description`, and `token duration`
-6. In the token type drop-down menu, you can select `Read Only`, `Full access` or custom permissions for the API (note: if setting a custom permission, please make sure to select `find` and `findOne`.
-7. API token will be displayed on clicking the “save” button
-8. Copy the token displayed (i.e. the token will be used in configuring `dlt` secrets)
-
-## Initialize the source with a default pipeline
-
-Initialize the source with a default with the following command:
-
-`dlt init strapi <destination>`
+Since the endpoints available in Strapi depend on your API setup created, you need to be aware of which endpoints you will be ingesting in order to get data from Strapi into your warehouse.
 
-For destination, you can use `bigquery`, `duckdb`, or one of the other [destinations](../general-usage/glossary.md#destination).
-
-Running this command will create a directory with the following structure:
-
-```python
-strapi_pipeline
-├── .dlt
-│   ├── .pipelines
-│   ├── config.toml
-│   └── secrets.toml
-├── strapi
-│   └── __init__.py
-│   └── readme.md
-│   └── strapi.py
-├── .gitignore
-├── strapi_pipeline.py
-└── requirements.txt
+## Initialize the pipeline
+```bash
+dlt init strapi bigquery
 ```
 
-## **Add credentials**
-
-In the `.dlt` folder, you will find `secrets.toml`, which looks like this:
-
-```python
-# put your secret values and credentials here. do not share this file and do not push it to github
-[sources.strapi]
-api_secret_key = "api_secret_key" # please set me up!
-domain = "domain" # please set me up!
-
-[destination.bigquery.credentials] # the credentials required will change based on the destination
-project_id = "set me up" # GCP project ID
-private_key = "set me up" # Unique private key (including `BEGINand END PRIVATE KEY`)
-client_email = "set me up" # Service account email
-location = "set me up" # Project location (e.g. “US”)
-```
+Here, we chose BigQuery as the destination. Alternatively, you can also choose redshift, duckdb, or any of the other [destinations.](https://dlthub.com/docs/dlt-ecosystem/destinations/).
```

Comment:
extra dot.
 [destinations.] and '/destinations/).'

Reply:
Yes, sorry, corrected.